src/main/java/dev/openfeature/javasdk/Value.java

```diff
@@ -19,7 +19,27 @@ public class Value {
     private final Object innerObject;
 
     public Value() {
-        this.innerObject = null; 
+        this.innerObject = null;
+    }
+
+    /**
+     * Construct a new Value with an Object.
+     * @param value to be wrapped.
+     * @throws InstantiationException if value is not a valid type
+     *      (boolean, string, int, double, list, structure, instant)
+     */
+    public Value(Object value) throws InstantiationException {
+        // integer is a special case, convert those.
+        this.innerObject = value instanceof Integer ? ((Integer)value).doubleValue() : value;
+        if (!(this.isNull()
+            || this.isBoolean()
+            || this.isString()
+            || this.isNumber()
+            || this.isStructure()
+            || this.isList()
+            || this.isInstant())) {
```

Comment:
We can apply DeMorgan's law here if anyone prefers the other way around. I never know which is more re-able.

Reply:
I think its slightly more readable with the negation distributed since it's a longer list. But  not a strong preference.

Reply:
done!